pkg/daemon/ceph/client/subvolumegroup.go

```diff
@@ -28,10 +28,19 @@ import (
 
 // CreateCephFSSubVolumeGroup create a CephFS subvolume group.
 // volName is the name of the Ceph FS volume, the same as the CephFilesystem CR name.
-func CreateCephFSSubVolumeGroup(context *clusterd.Context, clusterInfo *ClusterInfo, volName, groupName string) error {
+func CreateCephFSSubVolumeGroup(context *clusterd.Context, clusterInfo *ClusterInfo, volName, groupName string, svgSpec *cephv1.CephFilesystemSubVolumeGroupSpec) error {
```

Comment:
unit tests?

Reply:
We dont have any unit tests for the `subvolumegroup.go client helper functions that's why nothing was required to be modified.